packages/openapi-framework/src/types.ts

```diff
@@ -70,12 +70,14 @@ interface OpenAPIFrameworkArgs {
 
 export interface OpenAPIFrameworkAPIContext {
   basePath: string;
+  basePaths: BasePath[];
```

Comment:
can we just use `basePaths`?  We can always iterate over that for V2 in downstream projects like `(?:express|koa)-openapi` even if there's only 1 basePath in the array.

Reply:
Yeah I just wasn't sure if that would be cool since it would be a breaking change, but I suppose that is what semver is meant to help with.  I will try to re-push at some point today.

Reply:
right now `openapi-framework` hasn't had a major version, so we can make breaking changes to it's API without forcing breaking changes on downstream projects.